runtimes/eden/src/constants.rs

```diff
@@ -33,6 +33,9 @@ pub const NODL: Balance = 100_000_000_000;
 pub const MILLI_NODL: Balance = NODL / 1_000;
 pub const MICRO_NODL: Balance = MILLI_NODL / 1_000;
 pub const NANO_NODL: Balance = MICRO_NODL / 1_000;
+pub const DOLLARS: Balance = NODL / 100; // = 10 MILLI
+pub const CENTS: Balance = DOLLARS / 100; // = 100 MICRO
+pub const MILLICENTS: Balance = CENTS / 1_000; // = 100 NANO
```

Comment:
I'd like to stay off these variables since they are less clear than the MILLI/NANO/MICRO ones, that's why I had removed them